esp/src/eclwatch/nls/pt-br/hpcc.js

```diff
@@ -153,19 +161,23 @@
     FetchingData: "Buscando dados...",
     fetchingresults: "Buscando Resultados",
     File: "Arquivo",
+    FileCluster: "Archivo Cluster",
```

Comment:
8 more mistakes: archivo = arquivo. I speak english and portuguese every day and my brain gets them mixed up. There are 8 Archivos => Arquivos